README.md

```diff
@@ -74,3 +74,21 @@ In case there is an unexpected issue with this process you can test this process
 ## Learn More
 
 For additional information about the Bridges Project please refer to [parity-bridges-common](https://github.com/paritytech/parity-bridges-common) repository.
+
+## Docker
+
+Can be found in [dockerhub](https://hub.docker.com/repository/docker/wirednkod13/parity-bridges-ui).
+_NOTE: alter url once this is moved_
+
+To build the image run the:
+```
+docker build -t parity-bridges-ui:dev .
+```
+
+Now that image is built, container can start with the following command, which will serve our app on port 8080.
+```
+docker run --rm -it -p 8080:80 parity-bridges-ui:dev
+```
+
+(optional) The `--env-file` param can be added in the command above if you intend to alter the default [ENV params](https://github.com/paritytech/parity-bridges-ui/blob/master/.env).
+(e.g. `docker run --rm -it --env-file .env -p 8080:80 parity-bridges-ui:dev` )
```

Comment:
```suggestion
(e.g. `docker build --env-file .env -t parity-bridges-ui:dev ` )
```
I think the env file will not matter when the image is already built, `nginx` doesn't really care :)